bundle/cert-manager-operator/0.15.0-alpha.1/cert-manager-operator.v0.15.0-alpha.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    description: "x509 certificate management for Kubernetes"
+    createdAt: "2020-04-20T12:00:00Z"
+    support: "cert-manager team"
+    alm-examples: ' [{"apiVersion":"operator.cert-manager.io/v1alpha1","kind":"CertManager","metadata":{"name":"cert-manager","namespace":"cert-manager"},"spec":{}}]'
+    capabilities: Full Lifecycle
+    categories: Security
+    certified: 'false'
```

Comment:
Do we go back and edit this file once we get it certified using this CSV?

Reply:
let's certify this one since we now have a full go!